src/main/resources/mixins.kami.json

```diff
@@ -47,9 +47,11 @@
     "MixinRenderPlayer",
     "MixinStateImplementation",
     "MixinTileEntityBeacon",
-    "MixinTileEntityEnchantmentTableRenderer",
     "MixinTileEntitySignRenderer",
     "MixinVisGraph",
-    "MixinWorld"
+    "MixinWorld",
+    "MixinTileEntityRendererDispatcher",
+    "MixinMapItemRenderer",
+    "MixinParicleManager"
```

Comment:
Please keep the mixins in alphabet order

Reply:
Intellij's autoformat does this for you